### DISCUSSION TITLE
Field Relationship on TextInput?

### DISCUSSION THREAD

**Discussion by lutfifirdaus** [filamentphp/filament#3486] [Help] [answered]
Hello everyone

I'm recently learn how to use filament package, then when i try to make some project, i get some problem when using form builder inside edit page to fill field on relationship, i know on filament documentation mentioning that its only specific field type such 'Select', 'Repeater', etc that can pass relationship data. 

So does anyone know trick or solution how to pass relationship data to TextInput or other than fields that documentation specify?

**Comment by danharrin** [comment, marked as answer]
https://filamentphp.com/docs/2.x/forms/layout#saving-data-to-relationships

**Reply by lutfifirdaus** [reply]
thats what i looking, i miss that. Thanks for pointing me out!

**Comment by fsamapoor** [comment]
V3 docs:

https://filamentphp.com/docs/3.x/forms/advanced#saving-data-to-relationships

**Comment by vv12131415** [comment]
For some reason the answer does not work for 
`ViewAction::make()->form([])`

You cannot access `TextInput::make('realtionship.property')` on like you can do with `TextColumn::make('realtionship.property')`

**Reply by lutfifirdaus** [reply]
Have you tried to add `->record($this->model)` as mentioned in [here](https://filamentphp.com/docs/3.x/actions/prebuilt-actions/view#overview) ?

**Reply by Raza9798** [reply]
TextColumn can't be added into forms
